ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,249 @@
+﻿[[!meta title="Getting Involved"]]
+
+## How can I get involved in the OCF?
+
+Glad you asked! Students from all backgrounds are welcome to join staff. We
+don’t interview or have a selective process for joining—you simply have to
```

Comment:
```suggestion
don’t interview or have a selection process for joining—you simply have to
```

I think it is better to say we don't select rather than the process isn't selective (because that implies that there may be selection).